packages/types/src/deprecated.ts

```diff
@@ -0,0 +1,29 @@
+export type ModeOption = {
+  id: string;
+  selected?: boolean;
+  showTitle?: boolean;
+  text: JSX.Element;
+  title?: string;
+};
+
+export type ModeSelectorOptions = {
+  primary: ModeOption;
+  secondary?: ModeOption[];
+  tertiary?: ModeOption[];
+};
+
+export type ConfiguredMode =
+  | string
+  | {
+      mode: string;
+      label: string;
+      company?: string;
+    };
+
+export type ConfiguredModes = {
+  transitModes: ConfiguredMode[];
+  accessModes: ConfiguredMode[];
+  exlcusiveModes: ConfiguredMode[];
+  bicycleModes: ConfiguredMode[];
+  micromobilityModes: ConfiguredMode[];
+};
```

Comment:
This configuration objects are still in use in OTP-RR outside of mode selection form, so should they be deprecated?
- `accessModes`: in "My Account" (for wheelchair access), CallTaker
- `transitModes`: in form actions, CallTaker, itinerary narrative, stop marker
- `exclusiveModes`: in Calltaker